cleanlab/filter.py

```diff
@@ -60,21 +62,23 @@
     warnings.warn(w)
 
 
+T = TypeVar("T", bound=npt.NBitBase)
+
 def find_label_issues(
     labels: LabelLike,
-    pred_probs: np.ndarray,
+    pred_probs: npt.NDArray["np.floating[T]"],
     *,
     return_indices_ranked_by: Optional[str] = None,
     rank_by_kwargs: Optional[Dict[str, Any]] = None,
     filter_by: str = "prune_by_noise_rate",
     multi_label: bool = False,
     frac_noise: float = 1.0,
     num_to_remove_per_class: Optional[int] = None,
-    min_examples_per_class=1,
-    confident_joint: Optional[np.ndarray] = None,
+    min_examples_per_class: int=1,
+    confident_joint: Optional[npt.NDArray["np.floating[T]"]] = None,
     n_jobs: Optional[int] = None,
     verbose: bool = False,
-) -> np.ndarray:
+) -> npt.NDArray["np.floating[T]"]:
```

Comment:
I'd take a closer look at the docstring for the return value of `find_label_issues`.
It can be an array of boolean masks or integer indices.

Reply:
Yes, I am really sorry, I wasn't supposed to mark you for review just yet.

Reply:
No worries 👍, this was just something I noticed when you opened the draft PR.